pep-0517.txt

```diff
@@ -218,6 +241,14 @@ metadata. The directory passed in by the build frontend MUST be
 identical to the directory created by ``get_wheel_metadata``,
 including any unrecognized files it created.
 
+This method only places the *contents* of the wheel file, without actually
+creating the final artifact. This is to allow flexibility in how the result is
+going to be used. A front end like pip, which is only going to immediately
+install this wheel can speed up the installation process by skipping a round
+trip through zip. However a command like a hypothetical ``twine wheel`` could
```

Comment:
I think the point about the wheel cache is a good one. I'm skeptical of the benefit we'd get in pip from this. @dstufft knows more about this area than me, but I'd like to see the benefits more clearly spelled out (and specifically how we'd handle the wheel cache in this case).

I'm strongly -1 on having unpacked wheels in the cache and links. Not all filesystems support links, Windows has issues with them, dangling links could become something we need to consider, ... I can't see the benefits justifying this.